### PR TITLE
Added basic site navigation to the app

### PR DESCRIPTION
**Pull request by Anshul-Birla**
### Administrative Info
Monday Board ID: 853642866
Make sure your branch name conforms to: `<feature/staging/hotfix/...>/[username]/[Monday Item ID]-[3-4 word description separated by dashes]`. Otherwise, please rename your branch and create a new PR.

### Changes
What changes did you make?
- Added navbar to the website 

### Testing
How did you confirm your changes worked? 
- Tested clicking all the links

### Confirmation of Change 
Upload a screenshot, if possible. Otherwise, please provide instructions on how to see the change.

Capture

### CODE REVIEW

File: web/package.json [hunk 1]
```diff
@@ -3,8 +3,10 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "chokidar": "^3.5.1",
```
Comment (dacruz21) [inline web/package.json:6]: What's this for?

**Review by typokign — CHANGES_REQUESTED** [review]
Nice, excited to kick off the web app. A few things I found and a few conventions we should set now:

1. Let's put .css files in the same dir as their components, like App.css. Just makes it easier to discover and refactor them.
2. Let's replace the element selectors in CSS files with class selectors. Despite the React magic of being able to `import './foo.css'` in a component, the rules you import aren't actually scoped to just that file, they're applied globally. Try adding a `<ul>` to one of the pages, you'll see what I mean.
3. Can you remove the .gitkeep from components/ please? hacker roasted me for that in our last code review D:
4. Styling isn't 100% yet but I know you don't have the style guides, design team is working on that now, sorry for the delay!

While you're refactoring the CSS you _might_ want to look into [Styled Components](https://styled-components.com/). They're the new hot thing in react development - I haven't used them but if you want to see what the hype is about feel free to try them out.

File: web/package.json (hunk 1 above)
Comment (Anshul-Birla) [reply web/package.json:6]: if I dont npm install chokidar or w/e, the app doesnt compile. I googled the error and it said to just install the dependency and save it. Should I just remove it?

**Comment by Anshul-Birla** [comment]
> Nice, excited to kick off the web app. A few things I found and a few conventions we should set now:
> 
> 1. Let's put .css files in the same dir as their components, like App.css. Just makes it easier to discover and refactor them.
> 2. Let's replace the element selectors in CSS files with class selectors. Despite the React magic of being able to `import './foo.css'` in a component, the rules you import aren't actually scoped to just that file, they're applied globally. Try adding a `<ul>` to one of the pages, you'll see what I mean.
> 3. Can you remove the .gitkeep from components/ please? hacker roasted me for that in our last code review D:
> 4. Styling isn't 100% yet but I know you don't have the style guides, design team is working on that now, sorry for the delay!
> 
> While you're refactoring the CSS you _might_ want to look into [Styled Components](https://styled-components.com/). They're the new hot thing in react development - I haven't used them but if you want to see what the hype is about feel free to try them out.

I looked into Styled Components> They look interesting, but I feel like they add complexity to things, especially if there are animations and things we want to add. I kinda like having the styles and the component split into two things; it seems more natural with HTML/CSS.

File: web/package.json (hunk 1 above)
Comment (dacruz21) [reply web/package.json:6]: Hmm, looks like a library for file watching? I don't see it as a peerDependency of react-scripts but I suspect that's where it's coming from. Probably a bug, let me try to bump some versions and see if I can fix it...

Comment (dacruz21) [reply web/package.json:6]: gfdi I hate Facebook, here's the fix:

* Remove chokidar
* Bump react to ^17.0.2
* Bump react-scripts to 4.0.3 (NO carat cuz Facebook likes to publish breaking changes in patch versions :rage:)
* Manually add @ babel/core (no space, GitHub keeps trying to tag the babel user lol) to devDependencies, version ^7.13.0

Comment (dacruz21) [reply web/package.json:6]: Also change "jsx": "react" to "jsx": "react-jsx" in tsconfig.json for some reason

**Review by typokign — DISMISSED** [review]
Thanks! Let's do the styling in another PR so others can start to branch off of your work.

**Review by typokign — APPROVED** [review]
Looks great!

**Review by allisonbhavsar — APPROVED** [review]
looks great!

**Pull request merged**